### PR TITLE
Extend CUIGameCustom Lua export

### PR DESCRIPTION
**Pull request by ChugunovRoman**

### CODE REVIEW

File: src/xrGame/UIGameCustom_script.cpp [hunk 1]
```diff
@@ -13,6 +13,8 @@ SCRIPT_EXPORT(CUIGameCustom, (CDialogHolder), {
                          .def("wnd", &StaticDrawableWrapper::wnd),
 
         class_<CUIGameCustom, CDialogHolder>("CUIGameCustom")
+            .def("TopInputReceiver", &CUIGameCustom::TopInputReceiver)
```
Comment (Xottab-DUTY) [inline src/xrGame/UIGameCustom_script.cpp]: This function is already exported: https://github.com/OpenXRay/xray-16/blob/dev/src/xrGame/ui/UIWindow_script.cpp#L26

It should not be exported here.

File: src/xrGame/UIGameCustom_script.cpp [hunk 2]
```diff
@@ -13,6 +13,8 @@ SCRIPT_EXPORT(CUIGameCustom, (CDialogHolder), {
...
+            .def("SetMainInputReceiver", &CUIGameCustom::SetMainInputReceiver)
```
Comment (Xottab-DUTY) [inline src/xrGame/UIGameCustom_script.cpp]: This is wrong place for this export.

1. It should be exported here: https://github.com/OpenXRay/xray-16/blob/dev/src/xrGame/ui/UIWindow_script.cpp#L26
2. Maybe we also should export `SetTopInputReceiver`...

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]

File: src/xrGame/UIGameCustom_script.cpp (hunk 1 above)
Comment (ChugunovRoman) [reply src/xrGame/UIGameCustom_script.cpp]: Removed

File: src/xrGame/UIGameCustom_script.cpp (hunk 2 above)
Comment (ChugunovRoman) [reply src/xrGame/UIGameCustom_script.cpp]: 1. Moved
2. What's `SetTopInputReceiver`? Or did you mean rename `SetMainInputReceiver` to `SetTopInputReceiver`?

Comment (Xottab-DUTY) [reply src/xrGame/UIGameCustom_script.cpp]: I mean exporting additional function :)
```cpp
.def("SetMainInputReceiver", &CUIGameCustom::SetMainInputReceiver)
.def("SetTopInputReceiver", &CUIGameCustom::SetMainInputReceiver)
```
See here: https://github.com/OpenXRay/xray-16/pull/1019/files#r928311729

File: src/xrGame/ui/UIWindow_script.cpp [hunk 3]
```diff
@@ -23,6 +23,7 @@ SCRIPT_EXPORT(CDialogHolder, (CUIWindow), {
     [
         class_<CDialogHolder>("CDialogHolder")
             .def("TopInputReceiver", &CDialogHolder::TopInputReceiver)
+            .def("SetMainInputReceiver", &CDialogHolder::SetMainInputReceiver)
             .def("MainInputReceiver", &CDialogHolder::TopInputReceiver)
```
Comment (Xottab-DUTY) [inline src/xrGame/ui/UIWindow_script.cpp]:
```suggestion
            .def("TopInputReceiver", &CDialogHolder::TopInputReceiver)
            .def("MainInputReceiver", &CDialogHolder::TopInputReceiver)
            .def("SetTopInputReceiver", &CDialogHolder::SetMainInputReceiver)
            .def("SetMainInputReceiver", &CDialogHolder::SetMainInputReceiver)
```

File: src/xrGame/UIGameCustom_script.cpp (hunk 2 above)
Comment (ChugunovRoman) [reply src/xrGame/UIGameCustom_script.cpp]: Applied your suggestion

**Review by Xottab-DUTY — APPROVED** [review]

**Pull request merged**